build_tools/bazel_to_cmake/bazel_to_cmake_converter.py

```diff
@@ -371,6 +374,9 @@ def cc_test(self,
   def iree_runtime_cc_test(self, deps=[], **kwargs):
     self.cc_test(deps=deps + ["//runtime/src:runtime_defines"], **kwargs)
 
+  def iree_compiler_cc_test(self, deps=[], **kwargs):
+    self.cc_test(deps=deps + ["//compiler/src:defs"], **kwargs)
```

Comment:
Hmmm I think we should be encoding this in the CMake, not the Bazel->CMake (same for the runtime one above)